### ISSUE TITLE
Replace direct translations for VIP names with better localised alternatives

### ISSUE THREAD

**Issue by MandrakeHorse** [CorsixTH/CorsixTH#2636]
### What feature or enhancement would you like to add?

Our language files for Greek, Norwegian, Polish, Russian, Traditional Chinese and Simplified Chinese currently use direct translations for Theme Hospital's original VIP names. These could be replaced with more relevant and better localised alternatives, which can include national celebrities or politicians from those countries.

More pressingly, our Norwegian, Polish, Russian, Traditional Chinese and Simplified Chinese translations include the "Billy Savile OBE" reference, which should be replaced with our default substitute of "Sir Lancelot Spratt".

### How would this feature benefit the gameplay of CorsixTH?

Improved localisation will be appreciated by those who speak the above languages.

### Was the feature in the original game?

No

### What version of CorsixTH are you using?

0.68

### Does this feature/enhancement relate to a current problem?

No

**Comment by lewri** [comment]
I have checked to just make sure that new translators are being presented `Sir Lancelot Spratt` from the English, which they are. 

The replacement VIP was added ~2013 so it's likely those translations got made before the change.
https://github.com/CorsixTH/CorsixTH/commit/e66ba68f16a46e8b9ccdf5847b56212f3a50da7f

**Comment by MandrakeHorse** [comment]
I can create a pull request that replaces the Billy Savile strings. The rest of the strings can be localised separately.

**Comment by ARGAMX** [comment]
Nice proposal. 👍
Original names was absolutely no sense for russian player.
I have taken male names from top 50 russian celebrities. 
Changed them slightly, for 1 letter. Some in humour manner.
Here is my localized variant for russian:

vip_names = {
  [1] = "Хоккеист Александр Очечкин",
  [2] = "Мэр Сергей Обянин",
  [3] = "Артист Филипп Иркоров",
  [4] = "Музыкант Григорий Швепс",
  [5] = "Певец Валерий Целадзе",
  [6] = "Телеведущий Иван Шмургант",
  [7] = "Музыкант Николай Ебасков",
  [8] = "Актер Олег Табачков",
  [9] = "Писатель Борис Макунин",
  [10] = "Знаток Александр Груздь",
  health_minister = "Министр здравоохранения",
}

**Comment by Matroftt** [comment]
> ```
> vip_names = {
>   [1] = "Хоккеист Александр Очечкин",
>   [2] = "Мэр Сергей Обянин",
>   [3] = "Артист Филипп Иркоров",
>   [4] = "Музыкант Григорий Швепс",
>   [5] = "Певец Валерий Целадзе",
>   [6] = "Телеведущий Иван Шмургант",
>   [7] = "Музыкант Николай Жиробасков",
>   [8] = "Актер Олег Кабаков",
>   [9] = "Писатель Борис Макунин",
>   [10] = "Знаток Александр Груздь",
>   health_minister = "Министр здравоохранения",
> }
> ```

Some memetic persons should be added, ill suggest them soon

**Comment by MandrakeHorse** [comment]
> Nice proposal. 👍 Original names was absolutely no sense for russian player. I have taken male names from top 50 russian celebrities. Changed them slightly, for 1 letter. Some in humour manner. Here is my localized variant for russian:
...

Thank you for your suggested replacements, I've added them to my pull request and have credited you at the top of `russian.lua`.

**Comment by Matroftt** [comment]
```lua
 vip_names = {
  [1]  = "Мэр Нижнего Магнитошахтинска",
  [2]  = "Леонид Каневский",
  [3]  = '«Всезнающий» Онотоле Вассерман',
  [4]  = "Алексей Лесоповальный, представитель демократической оппозиции",
  [5]  = "Артист Иркоров ака «Babubei»",
  [6]  = "Телеведущий Иван Шмургант",
  [7]  = "Музыкант Николай Жиробасков",
  [8]  = "Актер Олег Кабаков",
  [9]  = "Писатель Борис Макунин",
  [10] = "Градоначальник Сергей Собякин",
  health_minister = "Министр здравоохранения",
}
```
Done, you can make changes to your PR

**Comment by ShimaKen523** [comment]
I would suggest following vip names for Japanese.
I picked some British famous people in Japan and some Japanese one, and changed few letters from actual name.
`vip_names = {
  [1] = "豊臣家康市長",
  [2] = "俳優ジェームズ・ポンド",
  [3] = "ハヤト=ミヤザキ監督",
  [4] = "英国女王エリガベス",
  [5] = "アバルート・アインシュイタン卿",
  [6] = "小泉進三郎 民主自由党代表",
  [7] = "チャーチル・サッチャー市議会議員",
  [8] = "元プロ野球選手 小谷翔平",
  [9] = "音楽家 ジャン・レノ",
  [10] = "J.K. ボウリング3世",
  health_minister = "保健省大臣",`

**Comment by ARGAMX** [comment]
> I would suggest following vip names for Japanese. I picked some British famous people in Japan and some Japanese one, and changed few letters from actual name. `vip_names = { [1] = "豊臣家康市長", [2] = "俳優ジェームズ・ポンド", [3] = "ハヤト=ミヤザキ監督", [4] = "英国女王エリガベス", [5] = "アバルート・アインシュイタン卿", [6] = "小泉進三郎 民主自由党代表", [7] = "チャーチル・サッチャー市議会議員", [8] = "元プロ野球選手 小谷翔平", [9] = "音楽家 ジャン・レノ", [10] = "J.K. ボウリング3世", health_minister = "保健省大臣",}`

Thanks. 
By the way, VIP in the game looks like a large mustachioed man in a hat.
And maybe it might be bit strange if his name will be "Queen of England"...

**Comment by MandrakeHorse** [comment]
@ShimaKen523 Thanks for providing some initial suggestions for Japanese VIP names! I have the following feedback:

1. Agree with @ARGAMX - since the VIP is always a man with a moustache and top hat, I don't feel that number 4 (英国女王エリガベス) or number 10 (J.K. ボウリング3世) would fit well.
2. In the English version of Theme Hospital/CorsixTH, number 9 is simply "a premiership footballer". From what I can tell, the Japanese equivalent of England's Premier League is the J1 League, so perhaps this string could be localised as "a J1 League footballer".

**Comment by lewri** [comment]
One of the VIPs in the English is actually female too. But likely due to the fact they were Burmese probably meant people didn't notice most of the time.

**Comment by TheCycoONE** [comment]
This got closed when the PR was merged but it seems there are many more suggestions listed here. Reopening at least until discussion and PRs wind down.

**Comment by ShimaKen523** [comment]
That's true. Considering the appearance in the game, men's name can be better.
I'd like to make them as followings, 5 from British, and 5 from Japanese.
`vip_names = {
 [1] = "豊臣家康市長",
 [2] = "俳優ジェームズ・ポンド",
 [3] = "ハヤト=ミヤザキ監督",
 [4] = "野々口英世",
 [5] = "アバルート・アインシュイタン卿",
 [6] = "小泉進三郎 民主自由党代表",
 [7] = "チャーチル・チャップリソ市議会議員",
 [8] = "音楽家 ジャン・レノ",
 [9] = "大リーガー 小谷翔平",
 [10] = "アイザック ニュートソ三世",
 health_minister = "保健省大臣",}`

**Comment by ARGAMX** [comment]
Is that task is actually done?
Can we finally close this task?

I was thinking, this task, in principle, seems to be completely endless.
After all, we won’t be able to translate it into all 22 languages we have.  
Plus, any new language we add could make this task relevant again for it. 
In short, it’s a never‑ending task. 

I’d rather add a note directly to the localisation files for the translator so that they don’t translate the names word for word, but think about some local, amusing names that are relevant to their region.

But I would open a separate task for this, with a clear understanding of when it should be closed as completed.
And I would close this task.

**Comment by tobylane** [comment]
I added a note to the wiki, though I agree with adding todos to the language files. 

> Translators are encouraged to add Very Important Person names that are relevant to the language. They should be male, humourous and inoffensive. As the original English includes roles like "A premiership footballer", similar descriptions may be included.
https://github.com/CorsixTH/CorsixTH/wiki/Localization#vip-names
